stylesheets/objects/_object.media.scss

```diff
@@ -21,6 +21,10 @@ a.media {
     }
 }
 
+.media--small .media__image {
+    font-size: 24px;
```

Comment:
Feels like we should prepend `.media__image` with `i` to ensure it's only set on icon fonts and not incorrectly on `img`'s.

`.media--small i.media__image`

Reply:
media image isn't always an icon